src/protocol/DataFeed.sol

```diff
@@ -14,8 +14,22 @@ contract DataFeed is IDataFeed {
     }
 
     /// @inheritdoc IDataFeed
-    function publish(uint256 numBlobs, bytes calldata data, MetadataQuery[] calldata queries) external payable {
-        uint256 nQueries = queries.length;
+    function publish(
+        uint256 numBlobs,
+        bytes calldata data,
+        HookQuery[] calldata metadataQueries,
```

Comment:
remove all occurances of "metadata"?

Reply:
isn't it actually returning the L1 metadata?